farms/static/css/snek.css

```diff
@@ -76,19 +71,19 @@ h1 {
   fill: var(--red);
 }
 
-.snek--3 .snektion .snek-body {
+.snek--curlyboi .snektion .snek-body {
```

Comment:
Ahhhhh! That makes a lot of sense, now. I noticed the `3`'s and such in the other JS file and was a little confused as to where it was coming from. This is more descriptive, thanks :)